<launch>
	<group ns="vision">
	<node pkg="stereo_webcam" type="stereo_node" name="driver">
		<param name="cameras" value="3"/>
		<param name="device0" value="/dev/video_left"/>
		<param name="device1" value="/dev/video_middle"/>
		<param name="device2" value="/dev/video_right"/>
		<param name="params0" value="file://$(find navi_bringup)/calibration/mono_l.yaml"/>
		<param name="params1" value="file://$(find navi_bringup)/calibration/mono_m.yaml"/>
		<param name="params2" value="file://$(find navi_bringup)/calibration/mono_r.yaml"/>

		<param name="auto_gain"  value="false"/>
		<param name="auto_white" value="false"/>
		<param name="hflip"      value="false"/>
		<param name="vflip"      value="false"/>
		<param name="width"      value="320"/>
		<param name="height"     value="240"/>
		<param name="nbuf"       value="10"/>
		<param name="fps"        value="10"/>
		<param name="frame"      value="/camera_link"/>

		<remap from="camera0" to="left"/>
		<remap from="camera1" to="middle"/>
		<remap from="camera2" to="right"/>
		<remap from="left/image"   to="left/image_raw"/>
		<remap from="middle/image" to="middle/image_raw"/>
		<remap from="right/image"  to="right/image_raw"/>
	</node>
	<node pkg="stereo_mux" type="mux_node" name="stereo_mux">
		<param name="fps" value="10"/>
		<param name="info_nl" value="file://$(find navi_bringup)/calibration/stereo_nl.yaml"/>
		<param name="info_nr" value="file://$(find navi_bringup)/calibration/stereo_nr.yaml"/>
		<param name="info_wl" value="file://$(find navi_bringup)/calibration/stereo_wl.yaml"/>
		<param name="info_wr" value="file://$(find navi_bringup)/calibration/stereo_wr.yaml"/>
	</node>
	</group>
</launch>
